DevoxxClientCommon/src/main/java/com/devoxx/model/SponsorBadge.java

```diff
@@ -0,0 +1,131 @@
+    public final StringProperty firstNameProperty() { return firstName; }
+    public final String getFirstName() { return firstName.get(); }
+    public final void setFirstName(String firstName) { this.firstName.set(firstName); }
+
+    private final StringProperty lastName = new SimpleStringProperty();
+    public final StringProperty lastNameProperty() { return lastName; }
+    public final String getLastName() { return lastName.get(); }
+    public final void setLastName(String lastName) { this.lastName.set(lastName); }
+
+    private final StringProperty company = new SimpleStringProperty();
+    public final StringProperty companyProperty() { return company; }
+    public final String getCompany() { return company.get(); }
+    public final void setCompany(String company) { this.company.set(company); }
+
+    private final StringProperty email = new SimpleStringProperty();
+    public final StringProperty emailProperty() { return email; }
+    public final String getEmail() { return email.get(); }
+    public final void setEmail(String email) { this.email.set(email); }
+
+    private final StringProperty details = new SimpleStringProperty();
+    public final StringProperty detailsProperty() { return details; }
+    public final String getDetails() { return safeStr(details.get()); }
+    public final void setDetails(String details) { this.details.set(details); }
+
+
+    // slug
+    private final StringProperty slug = new SimpleStringProperty(this, "slug");
+    public final StringProperty slugProperty() {
+       return slug;
+    }
+    public final String getSlug() {
+       return slug.get();
+    }
+    public final void setSlug(String value) {
+        slug.set(value);
+    }
+
+    @Override
+    public boolean contains(String keyword) {
+        if (keyword == null || keyword.isEmpty()) {
+            return false;
+        }
+        final String lowerKeyword = keyword.toLowerCase(Locale.ROOT);
+
+        return containsKeyword(getFirstName(), lowerKeyword) || 
+                containsKeyword(getLastName(), lowerKeyword) ||
+                containsKeyword(getCompany(), lowerKeyword)  ||
+                containsKeyword(getEmail(), lowerKeyword)    ||
+                containsKeyword(getDetails(), lowerKeyword)  ||
+                containsKeyword(getSlug(), lowerKeyword);
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        if (!super.equals(o)) return false;
+        SponsorBadge that = (SponsorBadge) o;
+        return Objects.equals(badgeId, that.badgeId) &&
+                Objects.equals(slug, that.slug);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(super.hashCode(), badgeId, slug);
+    }
+
+    public String toCSV() {
+        StringBuilder csv = new StringBuilder(super.toCSV());
+        csv.append(",").append(safeStr(getSlug()));
```

Comment:
I'm not sure if there's added value of providing the sponsor slug in the shared CSV? It's always going to be the same and the sponsor will most probably know who they are anyway.

Reply:
I added the slug to maintain proximity with the model.